src/platform/doc/index.rst

```diff
@@ -72,8 +72,16 @@ usually defined by the specific models and their documentation.
 * **Language Models**
     * `OpenAI's GPT`_ with `OpenAI`_ and `Azure`_ as Platform
     * `Anthropic's Claude`_ with `Anthropic`_ and `AWS Bedrock`_ as Platform
-    * `Meta's Llama`_ with `Azure`_, `Ollama`_, `Replicate`_ and `AWS Bedrock`_ as Platform
+    * `Meta's Llama`_ with `Azure`_, `Ollama`_, `Replicate`_, `AWS Bedrock`_ and `Groq`_ as Platform
+    * `Google's Gemma`_ with `Groq`_ as Platform
+    * `Mistral's Mixtral`_ with `Groq`_ as Platform
+    * `Alibaba's Qwen`_ with `Groq`_ as Platform
```

Comment:
Maybe we should use some kind of table for this?

Reply:
I agree, this doesn't scale well - not necessarily scope of this PR, but I'm in need for better ideas - even feels like a three dimensional table :laughing: